src/Components/Form/Datalist.js

```diff
@@ -0,0 +1,27 @@
+const CityDatalist = () => {
```

Comment:
I think it might be best practice to name the component and the function the same thing - i.e. name this component CityDataList